src/main/java/org/example/AccountInputService.java

```diff
@@ -0,0 +1,65 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class AccountInputService {
+    Scanner sc = new Scanner(System.in);
+    public AccountType getAccountType(){
+        System.out.print("""
+                1. Basic Banking Account
+                 Account can be opened with an initial/minimum deposit of Rs.1, 000.
+                2. Saving Account
+                 Account can be opened with an initial/minimum deposit of Rs.5,000
+                3. Current Account
+                 No Minimum Balance
+                Select Account Type:\s""");
+        int Type = sc.nextInt();
+        AccountType accountType;
+        if(Type==1){
+            accountType = AccountType.BASIC;
+        } else if (Type==2) {
+            accountType = AccountType.SAVING;
+        } else if(Type==3){
+            accountType = AccountType.CURRENT;
+        }else {
+            System.out.println("Invalid account type");
+            return getAccountType();
+        }
+        return accountType;
+    }
+    public int getAccountId(){
+        System.out.print("Enter account ID: ");
+        int accountId = sc.nextInt();
+        return accountId;
+    }
+    public String getAccountPassword(){
+        System.out.print("Enter Password: ");
+        String accountPassword = sc.next();
+        return accountPassword;
+    }
+    public CurrencyType getCurrency(){
+        System.out.print("""
+                1. USD
+                2. EUR
+                3. PKR
+                4. YEN
+                """);
+        System.out.print("Enter currency name: ");
+        String currency = sc.next().toUpperCase();
+        if(currency.equals("USD") || currency.equals("EUR") || currency.equals("PKR" ) || currency.equals("YEN")){
```

Comment:
don't use constants, use enum for condition value as well,
here don't use gpt, use slight of google

Reply:
try{
            return CurrencyType.valueOf(currency);
        }catch (Exception e){
            System.out.println("Invalid currency! Enter again your currency");
        }
        I don't use gpt or google I solved it myself I hope I solved right way

Reply:
good, also you removed the if conditions here,
so just think and tell which code is better and why? code with conditional statement or without conditional statement?

you need to remove the conditional statement here as well

int Type = sc.nextInt();
        AccountType accountType;
        switch (Type) {
        case 1:
                accountType = AccountType.BASIC;
                break;
            case 2:
                accountType = AccountType.SAVING;
                break;
            case 3:
                accountType = AccountType.CURRENT;
                break;
            default:
                System.out.println("Invalid account type");
                return getAccountType();
        }
        
   and you will take input in integer from user as 1,2, 3 as currently you are doing

Reply:
no doubt sir without conditional statements are much better than conditional statements

Reply:
yes sir i am currently working taking number accessing string value form enum